classes/robot/crawler.php

```diff
@@ -77,6 +77,7 @@ public function is_bot_valid() {
             return get_string('botcantgettestpage');
         }
         if ($result->redirect) {
+            return get_string('bottestpageredirected', 'local_linkchecker_robot') . $result->redirect;
```

Comment:
Please change this so that $result->redirect is passed into the lang string like this:

            return get_string('bottestpageredirected', 'local_linkchecker_robot', $result->redirect);

and then the lang string should format how the link appears